dockerfiles/buildall.sh

```diff
@@ -60,6 +60,13 @@ set -e
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 pushd $DIR 2>&1 > /dev/null
 
+. ../cmake_modules/parse_cmake.sh
+parse_cmake
```

Comment:
is this right?

Reply:
yes I think so (the .sh file defines a function...)